crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -88,14 +128,22 @@ pub(crate) async fn farm(
 
     let multi_farming = MultiFarming::new(
         MultiFarmingOptions {
-            base_directory,
+            base_directory: base_directory.clone(),
             client,
             object_mappings: object_mappings.clone(),
             reward_address,
             best_block_number_check_interval,
         },
         plot_size,
         max_plot_size,
+        move |plot_index, address, max_piece_count| {
```

Comment:
Is it an address or a public key of internal identity?

Reply:
`subspace_core_primitives::PublicKey`

Reply:
I mean does it serve the role of public key or reward address? If it is just a public key, then should be `public_key` variable to keep things consistent.

Reply:
Yeah, it is just a public key. I will change it, thanks :)